SPECS/mysql/mysql.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        MySQL.
 Name:           mysql
-Version:        8.0.33
+Version:        8.0.34
```

Comment:
We should close this PR in favor of:
- https://github.com/microsoft/CBL-Mariner/pull/6666